README.md

```diff
@@ -21,6 +21,7 @@ The AST explorer provides following code parsers:
 - HTML:
   - [htmlparser2][]
   - [parse5][]
+  - [angular][]
```

Comment:
Should this be capitalized?

```suggestion
  - [Angular][]
```

Reply:
These should be library names (kebab case). Should I change it to `@angular/compiler`?

Reply:
Sure!